### PR TITLE
feat(debian): add recipe for postgres

### PR DESCRIPTION
**Pull request by JakubKotkowiak**

### CODE REVIEW

File: recipes/newrelic/postgres/debian.yml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  - name: "NR_CLI_SSL"
+    prompt: "Use SSL to connect? (true/false)"
+  - name: "NR_CLI_CERT_AUTH_FILE"
+    prompt: "Certificate Authority file (if applicable)"
+  - name: "NR_CLI_TRUST_SERVER_CERTIFICATE"
+    prompt: "Trust certificate? (true/false)"
+  - name: "NR_CLI_CLIENT_CERT_FILE"
+    prompt: "Client Certificate file (if applicable)"
+  - name: "NR_CLI_CERT_KEY"
+    prompt: "PEM key file (if applicable)"
```
Comment (jbeveland27) [inline recipes/newrelic/postgres/debian.yml:51]: Need to do conditional logic based off the values for `NR_CLI_SSL` and `NR_CLI_TRUST_SERVER_CERTIFICATE` before writing to `postgresql-config.yml`. Details are outlined in #66.

File: recipes/newrelic/postgres/debian.yml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  version: "3"
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - taks: assert_pre_req
+        - task: setup
+
+    assert_pre_req:
```
Comment (jbeveland27) [inline recipes/newrelic/postgres/debian.yml:64]: Issue #66 specifies some output to the user that must be displayed. Need to add a `create_user_notification` similar to the [mongodb](https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/mongodb/rhel.yml#L74-L82) recipe with that criteria.

File: test/deploy/linux/postgres/install/debian/roles/configure/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    create_env_var: "false"
+  when: create_env_var is undefined
+
+- name: Install postgresql
+  apt:
+    name: ['postgresql']
+    update_cache: yes
+    state: latest
+  become: true
+
```
Comment (jbeveland27) [inline test/deploy/linux/postgres/install/debian/roles/configure/tasks/main.yml:21]: The rhel PR is doing some additional work to [init the database and run postgres](https://github.com/newrelic/open-install-library/blob/postgresRhel/test/deploy/linux/postgres/install/rhel/roles/configure/tasks/main.yml#L29-L39). Just wondering, is that not necessary here?

**Review by jbeveland27 — CHANGES_REQUESTED** [review]

File: test/deploy/linux/postgres/install/debian/roles/configure/tasks/main.yml (hunk 3 above)
Comment (JakubKotkowiak) [reply test/deploy/linux/postgres/install/debian/roles/configure/tasks/main.yml:21]: Not really no, it was necessary on rhel to make it work at all

**Review by jbeveland27 — APPROVED** [review]
LGTM 🚀

**Pull request merged**